roles/openshift_docker_facts/tasks/main.yml

```diff
@@ -49,7 +49,7 @@
   when: not openshift.common.is_containerized | bool
 
 - set_fact:
-    l_common_version: "{{ openshift.common.image_tag | default('0.0', True) | oo_image_tag_to_rpm_version }}"
+    l_common_version: "{{ openshift_version | default('0.0', True) | oo_image_tag_to_rpm_version }}"
```

Comment:
@detiber, in the end I created a new defaults file for openshift_docker_facts.  It should allow admins to override with the value as well as automatically set it based on openshift.docker.openshift_image_tag.